src/core/context/values.js

```diff
@@ -646,31 +634,33 @@ export class Values {
 /**
  * See `Values.scan()` method.
  *
- * @param {!./node.ContextNode} contextNode
- * @param {!ContextPropDef} prop
+ * @param {ContextNode} contextNode
+ * @param {IContextProp<?, ?>} prop
  * @return {boolean}
  */
 function scan(contextNode, prop) {
+  // @ts-ignore private access
   return contextNode.values.scan(prop);
 }
 
 /**
  * See `Values.scanAll()` method.
  *
- * @param {!./node.ContextNode} contextNode
+ * @param {ContextNode} contextNode
  * @param {?} unusedArg
- * @param {!Array<string>} state
- * @return {!Array<string>}
+ * @param {string[]} state
+ * @return {string[]}
  */
 function scanAll(contextNode, unusedArg, state) {
+  // @ts-ignore private access
```

Comment:
What is this doing? Is `values` not exposed on ContextNode?

Reply:
Correct, it's private

Reply:
Edit: `Values#scan` and `Values#scanAll` are `@protected`